src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1155,14 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes PREFIX(p/, d/, etc) from parameter string
      *
      * @param s  Parameter as a string
      * @param sign  Parameter sign to be removed
```

Comment:
Remember to modify all appropriate comments. Here, the "sign" should be updated.